config/application.rb

```diff
@@ -46,5 +46,6 @@ class Application < Rails::Application
     # Feedback URL (URL for feedback link in phase banner)
     # Use 'auto_add_path' for it to add a path link to the new_feedback route
     config.feedback_url = config.relative_url_root + '/feedback/new'
+    config.action_mailer.default_url_options = { host: 'example.com' } #TODO: this needs correcting
```

Comment:
It can be. I wanted to add a TODO and the settings.yml would have been unhappy. It can be fixed later i think